Makefile

```diff
@@ -13,6 +13,16 @@ demo-post:
 run-frontend:
 	@cd vue && npm run dev
 
+run-frontend-prod:
+	@echo "#############################"
+	@echo "# build vite for production #"
+	@echo "#############################"
+	@cd vue && npm run build
+	@echo "##########################"
+	@echo "# run vite in production #"
+	@echo "##########################"
+	@cd vue && npm run preview
+
```

Comment:
Should we include echo's for the other commands too? @lniehaus

Reply:
If the Makefile is only used from the remote container, `@echo` should be fine. `@echo ""` would lead to the output `""` under Windows.